src/workbooks/governance/workbook.json

```diff
@@ -1,8172 +1,8128 @@
-                },
-                {
-                  "id": "528e35b9-aca4-423f-9267-50f62011a3cb",
-                  "cellValue": "SelectedTab",
-                  "linkTarget": "parameter",
-                  "linkLabel": "Virtual machine",
-                  "subTarget": "RC_VM",
-                  "style": "link"
-                },
-                {
-                  "id": "7faacfc6-663e-4ff5-bb64-f86d995f9563",
-                  "cellValue": "SelectedTab",
-                  "linkTarget": "parameter",
-                  "linkLabel": "Storage + backup",
-                  "subTarget": "RC_Storage",
-                  "style": "link"
-                },
-                {
-                  "id": "c17ce2c0-83e6-4e5c-9c3e-f34cbf887e73",
-                  "cellValue": "SelectedTab",
-                  "linkTarget": "parameter",
-                  "linkLabel": "Network",
-                  "subTarget": "RC_Network",
-                  "style": "link"
-                },
-                {
-                  "id": "2f4e49d7-3198-4173-af1c-4cf4c5178000",
-                  "cellValue": "SelectedTab",
-                  "linkTarget": "parameter",
-                  "linkLabel": "PaaS",
-                  "subTarget": "RC_PaaS",
-                  "style": "link"
-                },
-                {
-                  "id": "f8f7e1fc-8f5d-442a-9788-3eabbf8ab275",
-                  "cellValue": "SelectedTab",
-                  "linkTarget": "parameter",
-                  "linkLabel": "Security",
-                  "subTarget": "RC_Security",
-                  "style": "link"
-                },
-                {
-                  "id": "80ad2db8-a21e-43e9-bd28-75d8d606eaf5",
-                  "cellValue": "SelectedTab",
-                  "linkTarget": "parameter",
-                  "linkLabel": "Monitoring",
-                  "subTarget": "RC_Monitoring",
-                  "style": "link"
-                },
-                {
-                  "id": "6fc0fef0-a016-4923-9239-b641eb5bdc4f",
-                  "cellValue": "SelectedTab",
-                  "linkTarget": "parameter",
-                  "linkLabel": "Services retirement",
-                  "subTarget": "RC_ServicesRetirement",
-                  "style": "link"
-                },
-                {
-                  "id": "e40dbf66-2abe-4bcf-acd7-1ee6d8fc950b",
-                  "cellValue": "SelectedTab",
-                  "linkTarget": "parameter",
-                  "linkLabel": "Resource age",
-                  "subTarget": "RC_Age",
-                  "style": "link"
-                },
-                {
-                  "id": "e112c6e1-db5e-4b0e-99e9-2edac0eba177",
-                  "cellValue": "SelectedTab",
-                  "linkTarget": "parameter",
-                  "linkLabel": "Tag explorer",
-                  "subTarget": "RC_Tag",
-                  "style": "link"
-                },
+    "version": "Notebook/1.0",
```

Comment:
@nteyan When you save this file, are you doing it in VS Code? The spacing shouldn't be changing.